tests/integ/test_session.py

```diff
@@ -344,3 +344,20 @@ def check(schema_name: str, expected_name: str) -> None:
     check("A", '"A"')
     check('"a b"', '"a b"')
     check('"a""b"', '"a""b"')
+
+
+def test_use_secondary_roles(session):
+    session.use_secondary_roles("all")
```

Comment:
This maybe an overkill but do we want to create enum for this similar to what we have in telemetry like https://github.com/snowflakedb/snowpark-python/blob/96e96a0fb431a32278d8a9463cd4ba0f1c28b91d/src/snowflake/snowpark/_internal/telemetry.py#L26 so that people don't make any mistakes regarding quoting.

Reply:
Yes I also think Enum is better here to avoid unexpected values - but it's a public API and previously we just use strings on the similar scenario for other APIs. Using enum here might also add complexity.

Reply:
Right. We have decided to not use Enum for all APIs at this time. Or else we would be inconsistent.
We can address this separately in the future.